core/templates/pages/exploration-player-page/templates/display-solution-modal.controller.spec.ts

```diff
@@ -56,7 +56,13 @@ describe('Display Solution Modal Controller', function() {
 
   importAllAngularServices();
 
-  beforeEach(angular.mock.module('oppia'));
+  beforeEach(angular.mock.module('oppia', function($provide) {
```

Comment:
Why are we providing it here? I don't see any changes to display-solution-modal.

Reply:
done